specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlServer.json

```diff
@@ -0,0 +1,1702 @@
+    "azure_auth": {
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "type": "oauth2",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/auditingSettings/{blobAuditingPolicyName}": {
+      "get": {
+        "operationId": "WorkspaceManagedSqlServerBlobAuditingPolicies_Get",
+        "summary": "Get server's blob auditing policy.",
+        "description": "Get a workspace managed sql server's blob auditing policy.",
+        "tags": [
+          "WorkspaceManagedSqlServer, BlobAuditing"
+        ],
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./workspace.json#/parameters/workspaceNameParameter"
+          },
+          {
+            "name": "blobAuditingPolicyName",
+            "in": "path",
+            "description": "The name of the blob auditing policy.",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "BlobAuditingPolicyName",
+              "modelAsString": true
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved workspace managed sql server's blob auditing policy.",
+            "schema": {
+              "$ref": "#/definitions/ServerBlobAuditingPolicy"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found.\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 409 UnsupportedWorkspaceType - Unsupported operation for the workspace type.\n\n * 500 InternalServerError - Something went wrong."
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Get blob auditing setting of workspace managed sql Server": {
+            "$ref": "./examples/GetWorkspaceManagedSqlServerBlobAuditingSettings.json"
+          }
+        }
+      },
+      "put": {
+        "operationId": "WorkspaceSManagedqlServerBlobAuditingPolicies_CreateOrUpdate",
```

Comment:
>WorkspaceSManagedqlServerBlobAuditingPolicies_CreateOrUpdate [](start = 24, length = 60)

Typo. It should be `WorkspaceManagedSqlServerBlobAuditingPolicies_CreateOrUpdate`

Reply:
resolved